core/os/os.cpp

```diff
@@ -373,8 +373,16 @@ OS::ScreenOrientation OS::get_screen_orientation() const {
 void OS::_ensure_data_dir() {
 
 	String dd = get_data_dir();
-	DirAccess *da = DirAccess::open(dd);
+	String current_dir;
+	DirAccess *da;
+	
```

Comment:
You have trailing spaces here and on line 398, it makes the style checks fail: https://travis-ci.org/godotengine/godot/jobs/266435497